cdap-proto/src/main/java/co/cask/cdap/proto/Id.java

```diff
@@ -67,8 +67,8 @@ private static boolean isValidNamespaceId(String name) {
     return namespaceMatcher.matchesAllOf(name);
   }
 
-  private String toString;
-  private int hashCode = -1;
```

Comment:
the -1 is depended on by this:
https://github.com/caskdata/cdap/blob/feature/4277/cdap-proto/src/main/java/co/cask/cdap/proto/Id.java#L112